test/unit/test-mysqlDao.js

```diff
@@ -1898,4 +1898,112 @@ describe('MysqlDao', function() {
         });
     });
 
+    describe('join', function() {
+
+        before(done => {
+            databaseSetup(err => {
+                if (err) return done(err);
+
+                const dummyUsers = [
+                    {id: 1, firstName: 'John', lastName: 'Doe', email: 'john.doe@gmail.com'},
+                    {id: 2, firstName: 'Jane', lastName: 'Doe', email: 'jane.doe@gmail.com'},
+                    {id: 3, firstName: 'James', lastName: 'Bond', email: 'lax4life-007@gmail.com'}
+                ];
+
+                const dummyTopSecretInfo = [
+                    {secretAgentCode: 1, handlerCode: null, intel: "It's gonna rain."},
+                    {secretAgentCode: 1, handlerCode: null, intel: "The sky is blue."},
+                    {secretAgentCode: 99, handlerCode: 2, intel: "Hello world."}
+                ];
+
+                // Create 500 pieces of high quality intel for James Bond.
```

Comment:
weird. but i like it. (this was the closest bitmoji i could find to bond...)

![bitmoji](https://render.bitstrips.com/v2/cpanel/10216054-127695570_21-s4-v1.png?transparent=1&palette=1&width=246)